### ISSUE TITLE
Please support all alias for busybox shell equally

### ISSUE THREAD

**Issue by ale5000-git** [koalaman/shellcheck#2978]
#### For new checks and feature suggestions
- [x] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/busybox bash
```
```sh
#!/bin/busybox ash
```
```sh
#!/bin/busybox sh
```

#### Here's what I wanted or expected to see:

There are just alias of the same busybox shell, a sort of ash (with some bash extension), even when called as `busybox sh`.
Only `busybox hush` is a different shell.

**Comment by grische** [comment]
The first two cases should be fixed with #2868
